x-pack/plugins/reporting/server/browsers/chromium/index.ts

```diff
@@ -15,3 +16,10 @@ export const chromium: BrowserDownload = {
   createDriverFactory: (binaryPath: string, captureConfig: CaptureConfig, logger: LevelLogger) =>
     new HeadlessChromiumDriverFactory(binaryPath, captureConfig, logger),
 };
+
+export const getChromiumDisconnectedError = () =>
```

Comment:
Exporting this as a function that returns an Error, so that when we need to throw the error, the stack trace makes sense.